sys/include/random.h

```diff
@@ -23,7 +23,8 @@
  *  - Musl C PRNG
  *  - Fortuna (CS)PRNG
  *  - Hardware Random Number Generator (non-seedable)
- *    Refer to your MCU manual for platform specific properties.
+ *    HWRNG differ in how they generate random numbers and may use PRNG internally.
```

Comment:
```suggestion
 *    HWRNG differ in how they generate random numbers and may not use a PRNG internally.
```

Reply:
(this is the list of available _PRNG_s after all, so that it may use an PRNG internally will surprise only a few readers ;-))